package.json

```
@@ -22,6 +23,7 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-tooltip": "^1.0.7",
+    "@shadcn/ui": "^0.0.4",
```

```suggestion
    -"@shadcn/ui": "^0.0.4",
```
**Remove @shadcn/ui package**
shadcn/ui is not a traditional package that we install via npm or yarn. Instead it's a collection of reusable components that we add to our project individually. So when you run `npx shadcn-ui@latest add input` The CLI adds the Input component files directly to your project in the components/ui directory. It shouldn't install a shadcn/ui package in your node_modules